lib/suse/connect/version.rb

```diff
@@ -1,6 +1,6 @@
 module SUSE
   # Provides access to version number of a gem
   module Connect
-    VERSION = '0.3.8'
+    VERSION = '0.3.9'
```

Comment:
You also need to update the RPM spec file. See https://github.com/SUSE/connect/blob/master/PACKAGE.md#step-3-update-package-version